src/containers/shared/components/Currency.tsx

```diff
@@ -11,10 +11,16 @@ export interface Props {
   shortenIssuer?: boolean
 }
 
+/*
+  LP token identifier is the identifier for LP tokens. All issued LP tokens start with 03 so we can use this to
+  make sure we're only converting actual hex values.
+ */
 const Currency = (props: Props) => {
   const { issuer, currency, link = true, shortenIssuer = false } = props
+  const LPTokenIdentifier = '03'
```

Comment:
Why do we not want to convert actual hex values?

Reply:
The hex converted will always be gibberish.

Reply:
Yea its necessary.